### PR TITLE
Update committee list

### PR DESCRIPTION
**Pull request by layerssss**

### CODE REVIEW

**Review by parndt — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -112,36 +113,41 @@ <h3>Meeting Minutes</h3>
 <section id="past-committee">
   <h2>Past Committee Members</h2>
   <ul>
+    <li>Ben Tillman (president 2022-2023)</li>
+    <li>Michael Yin (secretary 2021-2023)</li>
+    <li>Mark Haylock (general member 2021-2023)</li>
+    <li>Sarah Hay (general member 2022-2023)</li>
```
Comment (sarahhay7) [inline index.html]: This should be (general member 2021-2023, president 2023-2024)

**Review by sarahhay7 — CHANGES_REQUESTED** [review]
Just a couple of changes to the past committee details

File: index.html [hunk 2]
```diff
@@ -112,36 +113,41 @@ <h3>Meeting Minutes</h3>
...
+    <li>Matthew Wratt (general member 2022-2023</li>
```
Comment (sarahhay7) [inline index.html]: This is missing a closing bracket after 2023, and I think all of these should end with 2024, as the AGM was rounding off the end of the 2023-2024 time period. The current committee is for 2024-2025.

File: index.html [hunk 3]
```diff
@@ -112,36 +113,41 @@ <h3>Meeting Minutes</h3>
 <section id="past-committee">
   <h2>Past Committee Members</h2>
   <ul>
+    <li>Ben Tillman (president 2022-2023)</li>
```
Comment (sarahhay7) [inline index.html]: Ben was also president 2021-2022, so this should read (president 2021-2023, general member 2023-2024)

File: index.html [hunk 4]
```diff
@@ -112,36 +113,41 @@ <h3>Meeting Minutes</h3>
...
+    <li>Charlie Ablett (general member 2018-2019)</li>
+    <li>Samuel Williams (general member 2018-2019)</li>
+    <li>Amanda Wagener (general member 2017-2018 and 2014-2015, president 2016-2017, secretary 2015-2016)</li>
+    <li>Daniel Fone (secretary 2016-2018)</li>
+    <li>Patrick Davey (general member 2017-2018)</li>
+    <li>Max Hung (general member 2017-2018)</li>
+    <li>Raquel Moss (president 2017-2018, general member 2016-2017)</li>
+    <li>Merrin Macleod (secretary 2014-2015, general member 2016-2017, president 2020-2021)</li>
+    <li>Eoin Kelly (treasurer 2014-2016)</li>
+    <li>Philip Arndt (president 2014-2016, treasurer 2020-2021, treasurer 2022-2023)</li>
```
Comment (sarahhay7) [inline index.html]: The final phrase should be `treasurer 2022-2024`

File: index.html (hunk 1 above)
Comment (layerssss) [reply index.html]: yes, ditto.

File: index.html (hunk 2 above)
Comment (layerssss) [reply index.html]: Yes, Michael, Mark, Sarah, Mathew carried on to 2024, Only Ben handed over Presidency to Sarah (in 2023)

File: index.html (hunk 4 above)
Comment (layerssss) [reply index.html]: @sarahhay7 Yes, right. Also should I merge them, say `Philip Arndt (president 2014-2016, treasurer 2020-2021, 2022-2024)`?

Comment (layerssss) [reply index.html]: @sarahhay7 I guess there will be more previous members coming back doing the same role in the future as the year goes on.

Comment (layerssss) [reply index.html]: @sarahhay7 which will be nice.

Comment (sarahhay7) [reply index.html]: Yep, `Philip Arndt (president 2014-2016, treasurer 2020-2021, treasurer 2022-2024)` would be what I would put

**Review by sarahhay7 — APPROVED** [review]
LGTM

**Pull request merged**